### PR TITLE
#1826 Made `+inf.div` and `-inf.div` work with multiple arguments

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Closes: #1826 

What's done:
- Made `negative-infinity.div` work with multiple arguments
- Added tests
- Added todos

---

## PR-Codex overview
This PR implements a fix for the `div` method in `positive-infinity.eo` and `negative-infinity.eo`. 

### Detailed summary
- Implemented fix for `div` method in `positive-infinity.eo` and `negative-infinity.eo`
- Added support for multiple arguments in `div` method without using objects from eo-collections.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@Graur please have a look

File: eo-runtime/src/main/eo/org/eolang/positive-infinity.eo [hunk 1]
```diff
@@ -181,17 +181,94 @@
   #  worked with multiple arguments without using objects from eo-collections.
   #  See positive-infinity.plus and negative-infinity.plus as an example
```
Comment (Graur) [inline eo-runtime/src/main/eo/org/eolang/positive-infinity.eo:182]: @maxonfjvipon Do we need to remove this puzzle?

File: eo-runtime/src/main/eo/org/eolang/positive-infinity.eo [hunk 2]
```diff
@@ -181,17 +181,94 @@
...
+          term.gte 0 > @
+        [e]
+          term.gte 0.0 > @
+        FALSE
+
+    # @todo #1824:30min. Pass "is-nan-or-infinite" to "div-rec" here and in
+    #  negative-infinity. Now for some reason it's not possible to pass two or
+    #  more objects-functions to another object. To avoid code duplication
+    #  "is-nan-or-infinite" object should be passed to "times-rec" as well as
+    #  "is-term-gte-infinite"
```
Comment (Graur) [inline eo-runtime/src/main/eo/org/eolang/positive-infinity.eo:212]: @maxonfjvipon As far as I understand this todo issue, you want to provide to the object more than 3 arguments. If it's it right you can just add all your arguments to new array and pass this array as argument to particular object

**Review by Graur — CHANGES_REQUESTED** [review]

File: eo-runtime/src/main/eo/org/eolang/positive-infinity.eo (hunk 2 above)
Comment (maxonfjvipon) [reply eo-runtime/src/main/eo/org/eolang/positive-infinity.eo:212]: @Graur yes, I tried this way but check this out: https://github.com/objectionary/eo/issues/1841

File: eo-runtime/src/main/eo/org/eolang/positive-infinity.eo (hunk 1 above)
Comment (maxonfjvipon) [reply eo-runtime/src/main/eo/org/eolang/positive-infinity.eo:182]: @Graur yes, thanks

File: eo-runtime/src/main/eo/org/eolang/positive-infinity.eo (hunk 2 above)
Comment (Graur) [reply eo-runtime/src/main/eo/org/eolang/positive-infinity.eo:212]: @maxonfjvipon I see, let's keep todo

**Review by Graur — APPROVED** [review]
@yegor256 Please, have a look

**Comment by maxonfjvipon** [comment]
@Graur could you please review one more time and check what's wrong with windows build

**Comment by volodya-lombrozo** [comment]
> @Graur could you please review one more time and check what's wrong with windows build

@maxonfjvipon https://github.com/objectionary/eo/pull/1855

**Comment by maxonfjvipon** [comment]
@Graur please have a look one more time

**Review by Graur — APPROVED** [review]
@maxonfjvipon Looks good to me, thanks!
@yegor256 Please, have a look

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look

**Comment by maxonfjvipon** [comment]
@yegor256 please have a look

**Comment by maxonfjvipon** [comment]
@yegor256 reminder

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**